### ISSUE TITLE
widescreen support for LoD 1.09B

### ISSUE THREAD

**Issue by liPillON** [mir-diablo-ii-tools/SlashGaming-Diablo-II-Free-Resolution#128]
Hi, thank you for enabling users and modders to enjoy/enhance the game in new ways!
Would adding widescreen support for 1.09B be feasible?

I'm asking for this specific version because I'm a fan of the [Succulent](https://github.com/fearedbliss/Succulent#succulent-features) mod which is based on it.

I'd guess that since 1.09D is already supported, this might be relatively low-effort addition?

**Comment by Anonanon** [comment]
Also requesting support for 1.09b, there is demand out there for this version.

**Comment by IAmTrial** [comment]
It will take a while to find the necessary pieces to make this work. If addresses in 1.09B are slightly different, then that would basically require additional effort to locate 1.09B addresses. Do note that 1.09B support will be on a list of backlog items to complete.

**Comment by oli25** [comment]
1.09b is my favorite lod patch, since 1.09d doesn't add anything meaningful and yet breaks ctc. There are also some really good mods for 1.09b like Succulent or 1.09b final by Galaxyhaxz. I personally am also modding 1.09b so I would really love to see it supported.

**Comment by sharpwg1337** [comment]
Agreed this would be fantastic. If only now the D2GL author also made it compatible with 1.09B!
